config/sync/core.entity_form_display.node.page.default.yml

```diff
@@ -18,19 +21,34 @@ mode: default
 content:
   created:
     type: datetime_timestamp
-    weight: 5
+    weight: 8
     region: content
     settings: {  }
     third_party_settings: {  }
   field_branch:
     type: options_select
-    weight: 0
+    weight: 4
     region: content
     settings: {  }
     third_party_settings: {  }
+  field_display_titles:
+    type: boolean_checkbox
+    weight: 1
+    region: content
+    settings:
+      display_label: true
+    third_party_settings: {  }
+  field_hero_title:
+    type: string_textarea
+    weight: 2
+    region: content
+    settings:
+      rows: 2
+      placeholder: 'If empty, the regular page title will be shown..'
```

Comment:
```suggestion
      placeholder: 'If empty, the regular page title will be shown.'
```

Reply:
It was actually on purpose, as i generally add ".." in placeholder fields, but it's not a must for me, so i'll just change it

Reply:
I was wondering what this means. Doesn't the logic in the template either show the field_hero_title or nothing at all?

Reply:
> [...] i generally add ".." in placeholder fields

OK. I would expect either a `.` or a `...`